MediaPlayer/GStreamerMediaPlayer/src/MediaPlayer.cpp

```diff
@@ -47,6 +47,8 @@ using namespace avsCommon::utils::memory;
 using namespace avsCommon::utils::configuration;
 using MediaPlayerState = avsCommon::utils::mediaPlayer::MediaPlayerState;
 
+static const unsigned SAMPLE_RATE = 16000;
```

Comment:
SAMPLE_RATE is only used within the method MediaPlayer::WriterCallback, why not move the definition into that function? If there is some reason for declaring objects outside of the scope they are used in, that method is only visible if XMOS_AVS_TESTS is defined, so we could at least wrap it in that definition?